Demos/Blazorise.Demo/Pages/Tests/SignaturePadPage.razor

```diff
@@ -0,0 +1,125 @@
+                        <FieldLabel>Dot Size</FieldLabel>
+                        <FieldBody>
+                            <NumericPicker @bind-Value="@dotSize" />
+                        </FieldBody>
+                    </Field>
+                    <Field>
+                        <FieldLabel>Min Width</FieldLabel>
+                        <FieldBody>
+                            <NumericPicker @bind-Value="@minLineWidth" />
+                        </FieldBody>
+                    </Field>
+                    <Field>
+                        <FieldLabel>Max Width</FieldLabel>
+                        <FieldBody>
+                            <NumericPicker @bind-Value="@maxLineWidth" />
+                        </FieldBody>
+                    </Field>
+                    <Field>
+                        <FieldLabel>Throttle</FieldLabel>
+                        <FieldBody>
+                            <NumericPicker @bind-Value="@throttle" />
+                        </FieldBody>
+                    </Field>
+                    <Field>
+                        <FieldLabel>Min Distance</FieldLabel>
+                        <FieldBody>
+                            <NumericPicker @bind-Value="@minDistance" />
+                        </FieldBody>
+                    </Field>
+                    <Field>
+                        <FieldLabel>Velocity Filter Weight</FieldLabel>
+                        <FieldBody>
+                            <NumericPicker @bind-Value="@velocityFilterWeight" />
+                        </FieldBody>
+                    </Field>
+                    <Field>
+                        <FieldLabel>Read only</FieldLabel>
+                        <FieldBody>
+                            <Check @bind-Checked="@readOnly" />
+                        </FieldBody>
+                    </Field>
+                </Fields>
+            </CardBody>
+        </Card>
+    </Column>
+</Row>
+
+<Row>
+    <Column>
+        <Card Margin="Margin.Is4.OnY">
+            <CardHeader>
+                <CardTitle>Signature pad</CardTitle>
+            </CardHeader>
+            <CardBody>
+                <SignaturePad @bind-Value="@data" BackgroundColor="@backgroundColor" PenColor="@penColor" DotSize="@dotSize" MinLineWidth="@minLineWidth" MaxLineWidth="@maxLineWidth" Throttle="@throttle" MinDistance="@minDistance" VelocityFilterWeight="@velocityFilterWeight" ReadOnly="@readOnly" BeginStroke="@OnBegin" EndStroke="@OnEnd" />
+            </CardBody>
+        </Card>
+    </Column>
+
+    <Column>
+        <Card Margin="Margin.Is4.OnY">
+            <CardHeader>
+                <CardTitle>Preview</CardTitle>
+            </CardHeader>
+            <CardBody>
+                <Image Source="@Image64" />
+            </CardBody>
+        </Card>
+    </Column>
+</Row>
+
+
+
+@code {
+    byte[] data = null;
+
+    string Image64 => data != null ? $"data:image/png;base64,{Convert.ToBase64String( data )}" : null;
```

Comment:
Can we expose `private static string GetDataUrl( byte[] data, SignaturePadImageType imageType )`?